site/content/entry/telemetry-and-why-thats-a-good-thing.md

```diff
@@ -0,0 +1,59 @@
++++
+date = "2017-08-02T21:02:00-04:00"
+title = "Telemetry (And Why That's A Good Thing)"
+tags = [ "citra-release" ]
+author = "anodium"
+forum = 0000
++++
+
+Citra has some [issues](https://github.com/citra-emu/citra/issues), and by its
+nature as an open source project, they are visible to everyone and fixable by
+anyone. Unfortunately though, most contributions are made by
+[a small minority of developers](https://github.com/citra-emu/citra/graphs/contributors).
+These developers have found it difficult to prioritize their efforts, since the
+issues are mostly reported outside of the GitHub tracker, on the forums, on
+chats, and elsewhere. Not to mention that most of the reports are too vague, are
+missing vital information such as the logs or system info, or both.
+
+Because of this, the Citra team has put together a framework to report data about
+our users to our server, and use that data to discover what are the most popular
```

Comment:
Because of this, the Citra team has put together a framework to report data about how Citra is used to our server, ..."